x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts_list.ts

```diff
@@ -349,11 +349,11 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await deleteAlerts([createdAlert.id]);
     });
 
-    it.skip('should delete all selection', async () => {
+    it('should delete all selection', async () => {
       const namePrefix = generateUniqueKey();
       let count = 0;
       const createdAlertsFirstPage = await Promise.all(
```

Comment:
Since we're only creating one alert, we can cleanup some of the looping logic at the same time.

```suggestion
      const createdAlert = await createAlert({ name: `${namePrefix}-1` });
```